### DISCUSSION TITLE
Share & showcase

### DISCUSSION THREAD

**Discussion by kctekn** [kctekn/obsidian-TagsRoutes#17] [Show and tell]
Here, you can post a snapshot of an instance that reflects your vault structure, which might be visually appealing. And also maybe with the color scheme, display settings if you like.

**Comment by kctekn** [comment]
The default dark theme:
![default-dark-theme](https://github.com/user-attachments/assets/76354a71-9e89-4162-b182-2505c1acac50)

**Comment by kctekn** [comment]
The color scheme import from "Obsidian gruvbox" theme:
![import-from-Obsidian-gruvbox-theme](https://github.com/user-attachments/assets/afb52e62-e0cd-4f9c-8f40-38f2045a1b8c)

**Comment by AW2307** [comment]
I'll take the leap ;) 

Here's a screenshot of a section of my vault, which I focused into view by right-clicking select documents in the graph.

I find this feature to be among the most valuable, took a while to discover it.

<img width="452" alt="Screenshot 2024-11-05 at 05 39 50" src="https://github.com/user-attachments/assets/7de9ef8f-25e4-430d-9845-bdfccd4d5c77">
